bayesian/api/user_api.py

```diff
@@ -0,0 +1,70 @@
+"""Definition of all REST API endpoints for user management."""
+import uuid
+
+from flask import Blueprint, request
+from flask.json import jsonify
+
+from fabric8a_auth.auth import login_required
+from fabric8a_auth.errors import AuthError
+
+from bayesian.exceptions import HTTPError
+from bayesian.utility import user_utils
+from bayesian.utility.user_utils import UserException
+
+user_api = Blueprint('user_api', __name__, url_prefix='/user')
+
+
+@user_api.route('/readiness')
+def readiness():
+    """Handle the /readiness REST API call."""
+    return jsonify({}), 200
+
+
+@user_api.route('/liveness')
+def liveness():
+    """Handle the /liveness REST API call."""
+    return jsonify({}), 200
+
+
+@user_api.route('/<user_id>', methods=['GET'])
+@login_required
+def get_user(user_id):
```

Comment:
Can you create a API spec for these endpoints?

Reply:
Sure, will do

Reply:
https://app.swaggerhub.com/apis/prashbnair/User/1.0.0#/